staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1951,3 +1954,8 @@ type NetworkConfiguration struct {
 	PermitSlirpInterface              *bool  `json:"permitSlirpInterface,omitempty"`
 	PermitBridgeInterfaceOnPodNetwork *bool  `json:"permitBridgeInterfaceOnPodNetwork,omitempty"`
 }
+
+// GuestAgentPing signals the guest-agent based ping probe
+// +k8s:openapi-gen=true
+type GuestAgentPing struct {
+}
```

Comment:
Not sure if I sent a comment about this before.
We might want to put some value in here like enabled: bool just to make sure the api mechanics (also in other languages) don't drop this because it's empty.
Would like to know what others think, but I'm always a bit sceptical regarding empty structs on pointer vars as Kubernetes doesn't always have a stable nil, null, empty semantic yet.

Reply:
- We already use empty structs for other object, for ex. `rng` 
- `guestAgentPing: {}` seems quite self-explainable to me and `enabled: true` doesn't seem like the need here

Reply:
Don't mind changing this, but I feel it's simpler this way

Reply:
Alright, fine by me then :-)